### PR TITLE
Enable test specified docker image in grinder

### LINKED ISSUE #4561: criu-ubi-portable-checkpoint_test not able to test a specific checkpoint image
To test criu portable tests with cross platforms, we set the restore test to pull images from all available platforms, [see here](https://github.com/adoptium/aqa-tests/blob/08960bb0e08dfb0b463e16d86be3a938a89c99b1/external/external.sh#L334). It works well with nightly/weekly build, but for grinder test with a specific image link, it still tries to pull all platforms, and got error ([link](https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/32622/console)):
```
The named manifest is not known to the registry.
```

### PR DESCRIPTION
**Pull request by LongyuZhang**
- for grinder portable restore test, only pull the specified docker image
- for non-grinder pipeline, including `Grinder_imagePull`, pull all available platfrom checkpoint images
Closes #4561

### CODE REVIEW

File: external/external.sh [hunk 1]
```diff
@@ -329,12 +330,17 @@ if [ $command_type == "load" ]; then
 
 			restore_docker_image_name_list=()
 
-			echo "Testing images from nightly builds"
-			image_os_micro_architecture_list=($criu_combo_os_microarch_list)
-			for image_os_micro_architecture in ${image_os_micro_architecture_list[@]}
-			do
-				restore_docker_image_name_list+=("${docker_registry_url}/$job_name/${JDK_VERSION}-${JDK_IMPL}-${docker_os}-${platform}-${image_os_micro_architecture}:${build_number}")
-			done
+			if [[ $job_name == "Grinder" ]]; then
```
Comment (llxia) [inline external/external.sh:333]: `job_name` is a local variable. To make this simple, I think we should use `$JOB_NAME` directly and remove line 31

**Comment by llxia** [comment]
A similar change is needed for PingPerf. It can be in a separated PR. Eventually, we should merge the code for basic criu and PingPerf together.

File: external/external.sh (hunk 1 above)
Comment (LongyuZhang) [reply external/external.sh]: The PR has been updated, and issue related to PingPerf has been opened, which will resolve in another PR. Thanks.

**Review by llxia — APPROVED** [review]
LGTM

**Review by renfeiw — APPROVED** [review]
LGTM

**Pull request merged**